shared/profile/prove-enter-username-container.js

```diff
@@ -35,7 +35,7 @@ export default connector.connect(
     }
 
     return {
-      canContinue: profile.usernameValid,
+      canContinue: true,
```

Comment:
Nope... the validation occurs in store/reducer... this input bypasses that (uses state)

This makes sure canContinue is enabled and validation occurs when continue is clicked. I discussed with marco in person who agreed.